Source/WebKit/WebKitSwift/LinearMediaKit/LinearMediaPlayer.swift

```diff
@@ -526,11 +526,11 @@ extension WKSLinearMediaPlayer {
     public var isMutedPublisher: AnyPublisher<Bool, Never> {
         publisher(for: \.isMuted).eraseToAnyPublisher()
     }
-
+    #if !canImport(AVKit, _version: 1270)
```

Comment:
Nit: our style is to not indent compiler directives.
```suggestion
#if !canImport(AVKit, _version: 1270)
```